### ISSUE TITLE
Install issue

### ISSUE THREAD

**Issue by haydenbleasel** [fagbokforlaget/pdftohtmljs#4]
Hey,

Trying to install this module but for some reason it just won't install. `chmod` (line 9) implies it's a permissions issue, however `chmod 777 node_modules` didn't make a different. Also, it only seems to be this repo, all my other `npm install`s are working fine.

```
Haydens-MacBook-Pro:tracker haydenbleasel$ npm install pdftohtmljs --save
npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "pdftohtmljs" "--save"
npm ERR! node v0.12.1
npm ERR! npm  v2.7.3
npm ERR! path /Users/haydenbleasel/Projects/tracker/node_modules/pdftohtmljs/bin/pdftotextjs
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! enoent ENOENT, chmod '/Users/haydenbleasel/Projects/tracker/node_modules/pdftohtmljs/bin/pdftotextjs'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
```

**Comment by haydenbleasel** [comment]
Also tried `npm cache clean`, empty `.npmignore` and a few other solutions from [here](http://stackoverflow.com/questions/17990647/npm-install-errors-with-error-enoent-chmod)

**Comment by throrin19** [comment]
I have the same problem

**Comment by iapain** [comment]
Thanks for reporting this. It's been fixed in v0.3.7. There was a bug in package.json file
